concise-mid-tag.cddl

```diff
@@ -35,6 +34,34 @@ module-metadata-map = {
   * $$module-metadata-map-extension
 }
 
+module-entity-map = {
+  comid.entity-name => text,
+  ? comid.reg-id => any-uri,
+  comid.role => one-or-more<$module-role>,
+  * $$module-entity-map-extension
+}
+
+$module-role /= module-tag-creator
+$module-role /= module-creator
+$module-role /= module-distributor
+$module-role /= module-licensor
+$module-role /= module-maintainer
+
```

Comment:
I think these should go to the comid-code-points.cddl?

Reply:
Agree will change this!